xml/System.Diagnostics/Activity.xml

```diff
@@ -892,9 +909,9 @@ The root ID is a substring from the <xref:System.Diagnostics.Activity.Id> or the
         <Parameter Name="propertyValue" Type="System.Object" Index="1" FrameworkAlternate="net-5.0" />
       </Parameters>
       <Docs>
-        <param name="propertyName">To be added.</param>
-        <param name="propertyValue">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="propertyName">The name to associate the value with.</param>
+        <param name="propertyValue">The object to attach and map to the property name.</param>
+        <summary>Attaches any custom object to this activity. If the specified <paramref name="propertyName" /> was previously associated with another object, the property will be updated to be associated with the new <paramref name="propertyValue" /> instead.</summary>
```

Comment:
><summary>Attaches  [](start = 8, length = 18)

Could you please add a sentence like:

It is recommended to use a unique property name to avoid any conflict with anyone else use same value.